### DISCUSSION TITLE
Sensore Temp e Umidita per esterno

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#3267] [Hardware & Firmware]
Ciao a tutti, 
sto cercando un sensore temperatura e umidità da applicare all'esterno.
se possibile a batteria, cosa mi potete consigliare?
non mi volevo orientare sulle stazioni meteo, ma semplice sensore come utilizzo in casa.

**Comment by KC-inDomus** [comment]
[questo](https://indomus.it/componenti/xiaomi-mijia-aqara-sensore-di-temperatura-di-umidita-e-pressione/) va benissimo, puché lo mantieni lontano dalla pioggia.
